README.md

```diff
@@ -59,7 +59,7 @@ It is possible to create and interface with MyoSuite environments just like any
 
 
 ```python
-from myosuite.utils import gym
+from myosuite.utils.import_utils import gym
```

Comment:
this is pretty bad for the user as it is nested in utils or utils ... we should have the formulation with minimal depth of the import e.g. `from myosuite.utils import gym`

Reply:
SG